src/components/HomeInfo.js

```diff
@@ -1,16 +1,34 @@
 import React from 'react';
 import { Button } from '../atoms/Button';
 import { Avatar } from '../atoms/Avatar';
+import styled from 'styled-components';
+import { useResponsiveProps } from 'atomic-layout'
+
+const StyleTitle = styled.h1`
+font-size: ${({ fontSize }) => fontSize}px;
+`
+
+const Title = (props) => {
+    const { children, fontSize } = useResponsiveProps(props)
+    return <StyleTitle fontSize={fontSize}>{children}</StyleTitle>
+}
+
+
+const StyleMessage = styled.span`
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+`
 
 class HomeInfo extends React.Component {
     render() {
         return (
             <div className="container-homeInfo">
                 <h3>subtitle</h3>
-                <h1>title to tilorem</h1>
+                <Title fontSize={45} fontSizeMd={70}>title to tilorem</Title>
                 <div className="container-messager">
                     <Avatar />
-                    <span className="message">{this.props.children}</span>
+                    <StyleMessage className="message">{this.props.children}</StyleMessage>
```

Comment:
Why do you need to have a `message` class name?

Reply:
because I need   color: $color-grey;
I will change it later